### ISSUE TITLE
Feature Requests for Event API (more standardization) (gcode #222)

### ISSUE THREAD

**Issue by grgcombs** [openstates/openstates-scrapers#73]
Are all times still encoded as UTC or are they local times now?  I was looking at Utah and I thought I saw some meeting times were 3 or 4am once I localized them ...

Can we set the "link" field to use the first sources.url if there isn't one set already?

For states besides TX, can we scrape the meeting announcement text into notes wherever possible?

Ditto for scraping committee and chamber strings for the "participants" wherever possible (like in TX).

A big huge request would be to standardize on some kind of "title" for the event.  The "description" field varies greatly  from state to state.  Some include the type, date, and time in the string, others just have the title.  Would be great if this title had normalized capitalization (I'm staring at you, Kentucky!).

Basically, I'm trying to build a consistent, concise summary text for each event to display in a limited amount of space.  Ideally, I could get a lot of this for free from the API with a little more standardization on either the "description" field or the addition of a new "title" or "summary" field ...

"(S) Business & Commerce\n   When: 7/28/11 - 10:00 AM\n  Location: E1.016 (Hearing Room)"
"(H) Higher Education\n   When: 7/30/11 - 9:00 AM\n  Location: E2.543 - (CANCELED)"

 Alternatively, a "title" field could simply be "Business & Commerce" and I can cook up the rest from the other data you've inserted into the event dictionary.

Greg

---

http://code.google.com/p/openstates/issues/detail?id=222
